packages/core/lib/console.js

```diff
@@ -143,16 +126,53 @@ class Console extends EventEmitter {
     abstractions.forEach(abstraction => {
       contextVars[abstraction.contract_name] = abstraction;
     });
+  }
+
+  runSpawn(inputStrings, options, callback) {
+    let args;
+    if (typeof BUNDLE_CONSOLE_CHILD_FILENAME !== "undefined") {
+      args = path.join(__dirname, BUNDLE_CONSOLE_CHILD_FILENAME);
+    } else {
+      args = path.join(__dirname, "../lib/console-child.js");
+    }
+
+    const spawnOptions = { stdio: ["inherit", "inherit", "inherit"] };
 
-    this.repl.setContextVars(contextVars);
+    try {
+      //serialize config networks entries
+      const configNetworks = {};
+      for (const key in options.networks) {
+        configNetworks[key] = JSON.stringify(options.networks[key]);
+      }
```

Comment:
How many network properties are used in the child side? If it's 2-5 can this be changed to explicitly name them as arguments to spawnSync? A benefit of this approach would be to document what props are being used.
```
const networkProps = [ '--networkProp1', value1, '--networkProp2', 'value2'...];
SpawnSync("node", [...otherArgs, ...networkProps])
```